docs/manual/docs/cmd/teams/report/report-useractivitycounts.md

```diff
@@ -0,0 +1,26 @@
+# teams report useractivitycounts
+
+Get the number of Microsoft Teams activities by activity type. The activity types are team chat messages, private chat messages, calls, and meetings.
+
+## Usage
+
+```sh
+teams report useractivitycounts [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-p, --period <period>`|The length of time over which the report is aggregated. Supported values `D7, D30, D90, D180`
+`--verbose`|Runs command with verbose logging
```

Comment:
@sprider seems that the issue description has additional parameter, but it is missing in the command should we implement it or leave it for another iteration?
```
-d, --date [date] | The date for which you would like to view the users who performed any activity. Supported date format is YYYY-MM-DD.
```

Reply:
@VelinGeorgiev  date parameter is not required for this command. 

Please proceed with merge.

Reply:
@VelinGeorgiev I guess you are referring to issue #1029 by mistake.